server.js

```diff
@@ -91,7 +91,8 @@ middleware.forEach(func => app.use(func))
 // Set up App
 var appViews = extensions.getAppViews([
   path.join(projectDir, '/app/views/'),
-  path.join(projectDir, '/lib/')
+  path.join(projectDir, '/lib/'),
+  path.join(packageDir, '/lib/nunjucks')
```

Comment:
Is this necessary for more than just tests?

Reply:
Just for tests.

Reply:
I'll put a comment on that line.